requirements.txt

```diff
@@ -1,34 +1,21 @@
-alembic==1.5.4
-attrs==20.3.0
-autopep8==1.5.5
-blinker==1.4
-certifi==2020.12.5
-chardet==4.0.0
-click==7.1.2
-Flask==1.1.2
-Flask-Migrate==2.6.0
-Flask-SQLAlchemy==2.4.4
+alembic==1.8.1
+attrs==21.4.0
+click==8.1.3
+Flask==2.2.2
+Flask-Migrate==4.0.0
+Flask-SQLAlchemy==3.0.2
 gunicorn==20.1.0
-idna==2.10
 iniconfig==1.1.1
-itsdangerous==1.1.0
-Jinja2==2.11.3
-Mako==1.1.4
-MarkupSafe==1.1.1
-packaging==20.9
-pluggy==0.13.1
-psycopg2-binary==2.9.4
-py==1.10.0
-pycodestyle==2.6.0
-pyparsing==2.4.7
+itsdangerous==2.1.2
+Jinja2==3.1.2
+Mako==1.2.3
+MarkupSafe==2.1.1
+packaging==21.3
+pluggy==1.0.0
+py==1.11.0
+pyparsing==3.0.7
 pytest==7.1.1
-pytest-cov==2.12.1
-python-dateutil==2.8.1
-python-dotenv==0.15.0
-python-editor==1.0.4
-requests==2.25.1
-six==1.15.0
-SQLAlchemy==1.3.23
-toml==0.10.2
-urllib3==1.26.5
-Werkzeug==1.0.1
```

Comment:
Hmm I'm not sure what happened here, but somehow your list of packages got erased. I would recommend recommitting this file with the original list of packages